docs/resultTypes.md

```diff
@@ -351,4 +351,19 @@ interface section {
   line: string,
   anchor: string
 }
+```
```

Comment:
I think whole portion can be removed.

Reply:
o yes good catch. that must have slipped through.